src/Models/Services/Utils.ts

```diff
@@ -627,17 +627,32 @@ export const deleteModel = (id, data, models) => {
 };
 
 /**
- * Sort function to order items alphabetically. Case insensitive sort
+ * Sort function to order items from ascending or descending order, for boolean, numbers and strings. Case insensitive sort
  * NOTE: only works when property is one layer down
  * @param propertyName name of the property to sort on
- * @example listItems.sort(sortAlphabetically('textPrimary'))
+ *  @example listItems.sort(sortAscendingOrDescending('textPrimary'))
  * @returns Sort function to pass to `.sort()`
  */
-export function sortAlphabetically<T>(propertyName: keyof T) {
+export function sortAscendingOrDescending<T>(
```

Comment:
I think we should keep the old name (sortAlphabetically) but add the type of sort as second param.

Reply:
The thinking here was that it does more than just strings so the alphabetically name is misleading. I agree that the new name is very long. We could do with `defaultSort` or `sortDescending` and then it takes a prop whether it's ascending/descening.